packages/opentelemetry-node/lib/elastic-node-sdk.js

```diff
@@ -77,12 +79,15 @@ class ElasticNodeSDK extends NodeSDK {
         // TODO what `temporalityPreference`?
 
         // Disable metrics by config
-        const metricsDisabled = getEnvVar('ELASTIC_OTEL_METRICS_DISABLED');
+        const metricsDisabled = getEnvBoolean(
+            'ELASTIC_OTEL_METRICS_DISABLED',
+            false
+        );
         if (!metricsDisabled) {
             // Get metrics exporter protocol based on environment.
             const metricsExportProtocol =
                 process.env.OTEL_EXPORTER_OTLP_METRICS_PROTOCOL ||
-                getEnvVar('OTEL_EXPORTER_OTLP_PROTOCOL');
+                getEnvString('OTEL_EXPORTER_OTLP_PROTOCOL');
```

Comment:
```
% node --import @elastic/opentelemetry-node simple-http-request.js
...
{"name":"elastic-otel-node","level":40,"msg":"Metrics exporter protocol \"undefined\" unknown. Using default \"http/protobuf\" protocol","time":"2025-03-18T20:10:04.263Z"}
...
```

I think that log warn is because this doesn't have a default fallback now.